setup.py

```diff
@@ -17,12 +17,8 @@ def running_under_virtualenv():
     return False
 
 
-if os.environ.get('USE_SETUPTOOLS'):
```

Comment:
Do you have any context on why this was a thing? Because I don't, which means, I'm a bit afraid to just drop it.

Is distutils not a thing at all in python3?